packages/storage/src/prefixed_storage.rs

```diff
@@ -1,3 +1,5 @@
+use core::panic;
```

Comment:
Is this explicit import needed? I never imported it before?

Reply:
oh, that must have snuck in, probably an automatic import done by the IDE. i'll remove it.

Reply:
I don't know. It just something that rust-analyzer did. :shrug: